css/project-list-pages.styl

```diff
@@ -43,6 +43,10 @@
     .pill-button
       margin-right: 10px
       padding: 1em
+      color: rgb(64,64,64)
+      font-weight: normal
+      border-color: rgb(64,64,64) !important
+      box-shadow: rgb(64, 64, 64) 0px 0px 1px 0px inset !important
```

Comment:
Can we handle that separately? For now I'm just making the top pagination buttons look the same as the bottom, not adding any new style